02Lesson/01Project/index.html

```diff
@@ -1,9 +1,19 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>D's Dog Services</title>
+  <meta charset="utf-8">
 </head>
 <body>
-
+  <img src= "http://r.ddmcdn.com/s_f/o_1/cx_633/cy_0/cw_1725/ch_1725/w_720/APL/uploads/2014/11/too-cute-doggone-it-video-playlist.jpg" alt="smiling dog"/>
+  <h1 style=text-align:center>
+    Rain or Shine Dogs Don't Whine
+  </h1>
+  <h4>Services</h4>
+  <ul>
+    <li>Sitting</li>
+    <li>Walking</li>
+    <li>Transportation</li>
+  </ul>
 </body>
 </html>
```

Comment:
Please complete this assignment according to Homework specs to get full credit - https://web.austincodingacademy.com/_book/02Homework.html